chat_shell/chat_shell/prompts/attachments.py

```
@@ -0,0 +1,159 @@
+        Formatted prompt string for binary attachments, or empty string if none.
+    """
+    if not binary_attachments:
+        return ""
+
+    attachments_info = []
+    for att in binary_attachments:
+        att_id = att.get("id", "unknown")
+        filename = att.get("filename", "unknown")
+        mime_type = att.get("mime_type", "application/octet-stream")
+        download_url = att.get("download_url", f"/api/attachments/{att_id}/download")
+        file_ext = att.get("file_extension", "")
```

Ruff (0.14.13)

Local variable `file_ext` is assigned to but never used

Remove assignment to unused variable `file_ext`

(F841)